CMakeLists.txt

```diff
@@ -3,8 +3,17 @@
 
 #----- put platform independent settings in this section-----------------
 cmake_minimum_required(VERSION 2.8)
+if(POLICY CMP0054)
+  cmake_policy(SET CMP0054 NEW)
+endif()
 project (ROSE CXX C)
 
+OPTION(BUILD_SHARED_LIBS "Build all libraries shared" FALSE)
```

Comment:
I don't think you can right now.  The main rose library is done with a hard coded add_library(SHARED).